casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -705,6 +708,20 @@ class RuntimeManagerImpl[F[_]: Concurrent: Metrics: Span: Log](
        # }
        """.stripMargin('#')
 
+  // Executes deploy as user deploy with immediate rollback
+  // - InterpreterError is rethrown
+  def playExploratoryDeploy(term: String, hash: StateHash): F[Seq[Par]] = {
```

Comment:
We better have tests for this function

Reply:
I've created new ticket for tests.
https://rchain.atlassian.net/browse/RCHAIN-4018